.github/workflows/main.yml

```diff
@@ -45,5 +45,5 @@ jobs:
 
     services:
       materialized:
-        image: materialize/materialized:v0.26.3
+        image: materialize/materialized:v0.26.5
```

Comment:
Wish this was integrated in our main CI pipeline. Didn't bother bumping since we'll have to migrate the whole test suite next.